src/components/Figure/Image.js

```diff
@@ -46,78 +46,88 @@ const GalleryButton = ({ gallerySize, onClick }) => {
   )
 }
 
-class Image extends Component {
-  render() {
-    const {
-      src,
-      srcSet,
-      alt,
-      attributes = {},
-      maxWidth,
-      size: sizeProp,
-      aboveTheFold,
-      enableGallery = false,
-      gallerySize
-    } = this.props
+const Image = (props, context) => {
+  const {
+    src,
+    dark,
+    srcSet,
+    alt,
+    attributes = {},
+    maxWidth,
+    size: sizeProp,
+    aboveTheFold,
+    enableGallery = false,
+    gallerySize
+  } = props
 
-    const onClick = enableGallery
-      ? () => this.context.toggleGallery && this.context.toggleGallery(src)
-      : () => {}
+  const onClick =
+    enableGallery && context.toggleGallery
+      ? () => context.toggleGallery(src)
+      : undefined
 
-    const size = sizeProp || (sizeProp === undefined && imageSizeInfo(src))
-    const aspectRatio = size ? size.width / size.height : undefined
+  const size = sizeProp || (sizeProp === undefined && imageSizeInfo(src))
+  const aspectRatio = size ? size.width / size.height : undefined
```

Comment:
Because it already happened during testing, let's ensure we always pick the smaller and therefore higher `aspectRatio`:
```suggestion
  let aspectRatio = size ? size.width / size.height : undefined
  if (dark?.size) {
    aspectRatio = Math.min(aspectRatio, dark.size.width / dark.size.height)
  }
```

(we still don't support it and should leave the note in place but prevent overlap at the same time)